### DISCUSSION TITLE
Discussion about functionality and possible changes

### DISCUSSION THREAD

**Discussion by markrjones** [markrjones/listeo-package-handler#2] [Ideas]
Hi. Users who have already bought a package that has available listings can use it regardless of which category they are now creating a listing for. Example. In the past a user bought a cheap package for 3 listings. They now want to create a new listing in a category that we have created a package for, it's more expensive than the cheap on they already have. At present they are able to use on of the spare listings on their cheap package instead of being forced to pay for the new more expensive package. Is this an issue that we need to change the plugin to deal with differently?

**Comment by edindubai** [comment]
Hello, I had been working on modifying the class-listeo-core-submit for a while to make package selection come after choosing listing type and before the form, all almost worked except for the category dropdown not filtering properly while all other fields were correct. I then found your work as suggested by Gemini Advanced and maybe it's more elegant to put it last. Can you please update yours to use the most recent version of class-listeo-core-submit as it has had various changes since you did yours https://www.docs.purethemes.net/listeo/knowledge-base/changelog-listeo/ thanks

**Reply by markrjones** [reply]
Yes i can do that. I'll try to get to it soon.
